plugins/ltac/g_auto.mlg

```diff
@@ -64,10 +64,10 @@ let eval_uconstrs ist cs =
 
 let pr_auto_using_raw _ _ _  = Pptactic.pr_auto_using Ppconstr.pr_constr_expr
 let pr_auto_using_glob _ _ _ = Pptactic.pr_auto_using (fun (c,_) ->
-    let _, env = Pfedit.get_current_context () in
+    let _, env = Pfedit.get_current_context () in (* XXX *)
```

Comment:
@ppedrot are we sure we want the proof-local env here? In particular, won't the same term print differently depending on whether we take it as a `raw` or as a `glob` (since the printer for `raw` does not take the `env`)?

Reply:
That's only used in a few tactics, so you'll most likely never use the raw printer. `Print Ltac` uses the globalized one, raw printing would only be used by the beautifier.